XMonad/Actions/Search.hs

```diff
@@ -334,9 +340,9 @@ searchEngineF = SearchEngine
 
 -- The engines.
 alpha, amazon, arXiv, aur, clojureDocs, codesearch, cratesIo, deb, debbts, debpts, dictionary, duckduckgo, ebay, flora,
-  github, google, hackage, hoogle, images, imdb, lucky, maps, mathworld, ncatlab, openstreetmap, protondb, rosettacode, rustStd,
+  github, google, hackage, homeManager, hoogle, images, imdb, lucky, maps, mathworld, ncatlab, openstreetmap, protondb, rosettacode, rustStd,
   scholar, sourcehut, stackage, steam, thesaurus, vocabulary, voidpgks_x86_64, voidpgks_x86_64_musl, wayback, wikipedia, wiktionary,
-  youtube, zbmath, noogle :: SearchEngine
+  youtube, zbmath, nixos, noogle :: SearchEngine
```

Comment:
> The only thing I'd note is we keep the list of search engines in sorted order so it's easier to check if one you want is present.

I actually wanted to ask if you wanted me to put noogle (and nixos) in the right place now that I'm here changing this?